dashboard/app/models/script.rb

```diff
@@ -924,8 +923,40 @@ def self.setup(custom_files, new_suffix: nil, new_properties: {})
 
   # if new_suffix is specified, copy the script, hide it, and copy all its
   # levelbuilder-defined levels.
-  def self.add_script(options, raw_lesson_groups, raw_lessons, new_suffix: nil, editor_experiment: nil)
-    raw_script_levels = raw_lessons.map {|lesson| lesson[:scriptlevels]}.flatten
+  def self.add_script(options, raw_lesson_groups, new_suffix: nil, editor_experiment: nil)
+    raw_lessons = []
+
+    #recreate raw_lessons from raw_lesson_groups
+    raw_lesson_groups.each do |lesson_group|
```

Comment:
I'm excited to see that you went with this approach! I'm interested to hear at some point if it felt worth doing.

Reply:
Are we recreating `raw_lessons` because that's what's needed for the current code to work, or because we want to keep that for the long term? I think both are fine but I'm trying to understand what our strategy will be.

(I know we want to move away from the current seeding process long long term, but my impression is we'll probably have something similar to the current state for the next 6-12 months. Is this accurate?)

Reply:
Per earlier discussion i had with Dani, this is only here to let the current code in add_script keep working, while changing the code the calls it, as a way to break up this task into smaller pieces. 

Generally speaking, yes, the seed process will probably resemble it's current form for the next 6-12 months, except as changes to it are needed to unblock other work. We'll definitely want to tackle the seed problem head-on when we go to consolidate LB into prod.